mocha_test/initialParams.js

```diff
@@ -0,0 +1,26 @@
+import initialParams from '../lib/internal/initialParams';
+var expect = require('chai').expect;
+
+describe('initialParams', function() {
+  it('should curry any bound context to the wrapped function', function() {

```

Comment:
Also, I don't think "curry" is the right term here -- this is just context binding.